18-minimal/Dockerfile.c9s

```diff
@@ -0,0 +1,66 @@
+FROM ubi9/ubi-minimal
+
+EXPOSE 8080
+
+# Add $HOME/node_modules/.bin to the $PATH, allowing user to make npm scripts
+# available on the CLI without using npm's --global installation mode
+# This image will be initialized with "npm run $NPM_RUN"
+# See https://docs.npmjs.com/misc/scripts, and your repo's package.json
+# file for possible values of NPM_RUN
+# Description
+# Environment:
+# * $NPM_RUN - Select an alternate / custom runtime mode, defined in your package.json files' scripts section (default: npm run "start").
+# Expose ports:
+# * 8080 - Unprivileged port used by nodejs application
+ENV APP_ROOT=/opt/app-root \
+    # The $HOME is not set by default, but some applications need this variable
+    HOME=/opt/app-root/src \
+    NPM_RUN=start \
+    PLATFORM="el9" \
+    NODEJS_VERSION=18 \
+    NPM_RUN=start \
+    NAME=nodejs
+
+ENV SUMMARY="Minimal image for running Node.js $NODEJS_VERSION applications" \
+    DESCRIPTION="Node.js $NODEJS_VERSION available as container is a base platform for \
+running various Node.js $NODEJS_VERSION applications and frameworks. \
+Node.js is a platform built on Chrome's JavaScript runtime for easily building \
+fast, scalable network applications. Node.js uses an event-driven, non-blocking I/O model \
+that makes it lightweight and efficient, perfect for data-intensive real-time applications \
+that run across distributed devices." \
+    NPM_CONFIG_PREFIX=$HOME/.npm-global \
+    PATH=$HOME/node_modules/.bin/:$HOME/.npm-global/bin/:$PATH
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="Node.js $NODEJS_VERSION Micro" \
+      io.openshift.expose-services="8080:http" \
+      io.openshift.tags="builder,$NAME,${NAME}${NODEJS_VERSION}" \
+      io.openshift.s2i.scripts-url="image:///usr/libexec/s2i" \
+      io.s2i.scripts-url="image:///usr/libexec/s2i" \
+      com.redhat.dev-mode="DEV_MODE:false" \
+      com.redhat.deployments-dir="${APP_ROOT}/src" \
+      com.redhat.dev-mode.port="DEBUG_PORT:5858" \
+      com.redhat.component="${NAME}-${NODEJS_VERSION}-minimal-container" \
+      name="ubi9/$NAME-$NODEJS_VERSION-minimal" \
```

Comment:
```suggestion
      name="sclorg/$NAME-$NODEJS_VERSION-minimal-c9s" \
```

Reply:
Is this true? Do we really want to support NodeJS 18 - minimal for CentOS Stream 9? If not, then I guess, this Dockerfile can disappear.

Reply:
I don't know, maybe not necessarily. It's excluded now, so won't be built. I thought it might stay there for the case we want it at some point. Or do you think it should be better removed?